tests/jax/test_mnist.py

```diff
@@ -1,227 +0,0 @@
-# Copyright (c) 2022-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
The CI run the JAX example. This is great.
If a user or a dev run TE tests, I think currently the examples won't be run.
Any idea how to make them also run?

Reply:
It should work. User can follow `qa/L0_jax_unittest/test.sh` to run both unittests and examples

Reply:
The normal workflow to run tests is to run:
`pytest directory`

So I'm sure some users will skip the documentation on how to run the tests.
If by chance you have an idea how to fix that, that is great. Otherwise, it can stay like this. Check if there is a section on how to run tests and update it.

Reply:
> TE already document this?

No, it doesn't

Reply:
It needs to open another thread for discussion.